### PR TITLE
Tf fix ephemeral ip example

### PR DESCRIPTION
**Pull request by msampathkumar**
Use static ip for xlb forwarding rule …
6eda355
- Use `default` for all tf resources name, unless required
- Add global_address resource for static ip
- Update forwarding_rule to use static ip

If this PR is for Terraform, I acknowledge that I have:

- [x ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [x ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [ x] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ x] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none
- Ephemeral IPs are subjected to refresh after restart, so using static IP for HTTP External LB Example.

```

Also is the linked [sub-section- `#generating-downstream-tools`](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools) in readme page, does not seem to exist. Please check!!

### CODE REVIEW

File: mmv1/products/compute/terraform.yaml [hunk 1]
```diff
@@ -901,10 +901,11 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     examples:
       - !ruby/object:Provider::Terraform::Examples
         name: "external_http_lb_mig_backend_custom_header"
-        primary_resource_id: "google_compute_global_forwarding_rule"
+        primary_resource_id: "default"
         vars:
           xlb_network_name: "l7-xlb-network"
           backend_subnet_name: "l7-xlb-subnet"
+          address_name: "l7-xlb-static-ip"
```
Comment (rileykarson) [inline mmv1/products/compute/terraform.yaml:908]: We'll need to set `ignore_read_extra` on this example (https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/provider/terraform/examples.rb#L117-L119) against the `ip_address` field, as the API returns the address value (ip) when specified with the address id.

Comment (msampathkumar) [reply mmv1/products/compute/terraform.yaml:908]: Hi @rileykarson  - Thank you for your comments! As suggested , will add `ip_address` into ignore read extra & check tests.

**Comment by msampathkumar** [comment]
working on it !

**Comment by msampathkumar** [comment]
We can't any terraform related errors from our side. Please check and suggest!

**Review by rileykarson — APPROVED** [review]

**Comment by rileykarson** [comment]
Those failures are unrelated!

**Pull request merged**